schematic/models/validate_attribute.py

```diff
@@ -58,6 +59,7 @@ def generate_schema_error(
             error_col=attribute_name,
             error_message=error_message,
             error_val=invalid_entry,
+            message_level="error",
```

Comment:
why the message level here needs to be hard-coded as "error"?

Reply:
Because this is a helper to raise a message for a JSON Schema Validation Error, so those are always logged at the error level.